lib/signature.js

```diff
@@ -138,15 +139,26 @@ exports.verify = async ({event}) => {
     id: publicKeyId,
     assertionMethod: publicKeyId,
   };
-  const key = new Ed25519VerificationKey2018(_getPublicKey(publicKeyId));
+  const key = new Ed25519VerificationKey2020(_getPublicKey(publicKeyId));
+
+  ///////////////////// FIXME: The signer shouldn't have to be specified
+  key.signer = {
+    sign: () => {
+      console.log(
+        'ERROR: MergeEventEd25519Signature2020.signer called during verify().');
+      return 'INVALID';
+    }
+  };
+  /////////////////////
```

Comment:
If this code isn't here, the code throws creating the suite for verification. The Ed25519Signature2020 tries to create a signer even when a private key isn't supplied. What's up with that? /cc @dmitrizagidulin

Reply:
+1 agree that it shouldn't be needed.